content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -19,6 +19,9 @@ further_reading:
     - link: 'tracing/visualization/'
       tag: 'Documentation'
       text: 'Explore your services, resources, and traces'
+    - link: 'https://github.com/DataDog/dd-trace-dotnet/tree/master/samples'
+      tag: 'GitHub'
+      text: 'Examples of custom instrumentation'
```

Comment:
We should consider adding this to the setup pages, too, since some of the sample show how to set up automatic instrumentation in Docker:
```
/content/en/tracing/setup_overview/setup/dotnet-framework.md
/content/en/tracing/setup_overview/setup/dotnet-core.md
```